fix(apple): Disabling crashes disables watchdog (#6093)
getsentry/sentry-docs: src/platforms/apple/common/configuration/watchdog-terminations.mdx

    options.enableWatchdogTerminationTracking = NO;
}];
```

> If you disable the `enableCrashHandler` option, the SDK disables the watchdog termination tracking as well, cause
> the SDK would report false positive watchdog terminations for each crash.

Reviewer: ```suggestion
If you disable the `enableCrashHandler` option, the SDK will disable watchdog termination tracking. This will prevent false positive watchdog termination reporting for every crash.
```
